e2e/snapshot-escape/__tests__/__snapshots__/snapshot.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
these shouldn't be check in - probably aborted e2e tests which was unable to clean up after itself.

should be possible to run e.g. `yarn jest snapshot -u` and have it cleaned up